docs/faqs.md

```diff
@@ -63,7 +63,7 @@ Unless they are a general and extended utility in recipes (in which case, we sho
 
 ## What version should packages use for libraries without official releases?
 
-The notation shown below is used for publishing packages where the original library does not make official releases. Thus, we use a format which includes the datestamp corresponding to the date of a commit: `cci.<YEAR MONTH DAY>`. In order to create reproducible builds, we also "commit-lock" to the latest commit on that day. Otherwise, users would get inconsistent results over time when rebuilding the package. An example of this is the [RapidJSON](https://github.com/Tencent/rapidjson) library, where its package reference is `rapidjson/cci.20200410` and its sources are locked the latest commit on that date in [config.yml](https://github.com/conan-io/conan-center-index/blob/master/recipes/rapidjson/config.yml#L4). The prefix `cci.` is mandatory to distinguish as a virtual version provided by CCI. If you are interested to know about the origin, please, read [here](https://github.com/conan-io/conan-center-index/pull/1464).
+The notation shown below is used for publishing packages where the original library does not make official releases. Thus, we use a datestamp corresponding to the date of a commit. In order to create reproducible builds, we also "commit-lock" to the latest commit on that day. Otherwise, users would get inconsistent results over time when rebuilding the package. An example of this is the [NanoRange](https://github.com/tcbrindle/NanoRange) library, where its package reference is `nanorange/20191001` and its sources are locked the latest commit on that date: https://github.com/conan-io/conan-center-index/blob/master/recipes/nanorange/all/conandata.yml#L2L3
```

Comment:
I think this PR shouldn't modify this FAQ.

Reply:
What?! Probably I mixed the branches! I gonna fix it, thanks